unstract/connectors/src/unstract/connectors/databases/redshift/redshift.py

```diff
@@ -76,12 +76,12 @@ def sql_to_db_mapping(value: str) -> str:
         """
         python_type = type(value)
         mapping = {
-            str: "SUPER",
+            str: "VARCHAR(65535)",
```

Comment:
@kirtimanmishrazipstack QQ: Why this is fixed to 65535 ?

Reply:
@vishnuszipstack as mentioned  https://docs.aws.amazon.com/redshift/latest/dg/r_Character_types.html, max data redshift can hold is VARCHAR(65535) or 4KB. When I made to super, it was passing null value when insertion is made for large data set. 

Using  VARCHAR(65535), if data size is more than 4KB, we are throwing proper error

Reply:
@kirtimanmishrazipstack why it was passing null value when inserting larger data set with SUPER type. ?. as my understanding SUPER type supports 16M size https://docs.aws.amazon.com/redshift/latest/dg/r_SUPER_type.html cc: @vishnuszipstack

Reply:
@kirtimanmishrazipstack since it isn't immediately obvious, consider adding a comment if its still `VARCHAR`

Reply:
@kirtimanmishrazipstack I want to know why we can't use SUPER type. it supports 16M size?

Reply:
@jaseemjaskp as discussed, lets make it as varchar and show the error message to user